presto-parser/src/main/java/com/facebook/presto/sql/tree/DereferenceExpression.java

```diff
@@ -79,12 +79,26 @@ public static QualifiedName getQualifiedName(DereferenceExpression expression)
         return parts == null ? null : QualifiedName.of(parts);
     }
 
+    public static Expression from(QualifiedName name)
```

Comment:
Need to adapt from a qualified name to a chain of DereferenceExpression/Identifier.